e2e/ansible/playbooks/hyperconverged/test-k8s-volume-taints/test-volume-taints.yml

```diff
@@ -0,0 +1,245 @@
+         register: result
+         until: "'Running' in result.stdout"
+         delay: 120
+         retries: 5
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Copy the volume-policies yaml to kube master
+         copy:
+           src: "{{ volume_policy }}"
+           dest: "{{ result_kube_home.stdout }}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Replacing the key in volume-policies yaml
+         replace:
+           path: vol-policies.yaml
+           regexp: 'key: jk'
+           replace: 'key: {{ pvc_key }}'
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Replacing the value in volume-policies yaml
+         replace:
+           path: vol-policies.yaml
+           regexp: 'value: jv'
+           replace: 'key: {{ pvc_value }}'
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: deploy the volume-policies yaml
+         shell: source ~/.profile; kubectl apply -f "{{ volume_policy }}"
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+
+       - name: download test-pvc file
+         get_url:
+           url: https://raw.githubusercontent.com/openebs/openebs/master/e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool/test-pvc.yaml
+           dest: "{{ result_kube_home.stdout }}"
+           force: yes
+         register: result
+         until:  "'OK' in result.msg"
+         delay: 5
+         retries: 3
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+  
+       - name: Replacing the storage class name in test-pvc yaml
+         replace:
+           path: test-pvc.yaml
+           regexp: 'storageClassName: openebs-standard'
+           replace: 'storageClassName: openebs-taints-0.6.0'
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Create  namespace for PVC
+         shell: kubectl create ns "{{ namespace }}"
+         args:
+           executable: /bin/bash
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         ignore_errors: true
+     
+
+       - name: deploy the test-pvc yaml
+         shell: source ~/.profile; kubectl apply -f test-pvc.yaml -n "{{ namespace }}"
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Confirm volume container is running
+         shell: source ~/.profile; kubectl get pods --all-namespaces | grep pvc | grep {{item}} | grep Running | wc -l
+         args:
+           executable: /bin/bash
+         register: result
+         until: result.stdout|int >= 1
+         delay: 30
+         retries: 5
+         with_items:
+           - ctrl
+           - rep
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         
+ 
+       - name:
+         shell: source ~/.profile; kubectl get pods --all-namespaces -o wide | grep rep
```

Comment:
Please use the volume replica deployment name to obtain the list of replicas and their nodes